tools/cloud-build/daily-tests/ansible_playbooks/test-lustre-slurm.yml

```diff
@@ -16,19 +16,24 @@
 
 - name: Create output folder
   ansible.builtin.shell: |
-    [[ ! -d {{ custom_vars.output_dir }} ]] && sudo mkdir -m a+w {{ custom_vars.output_dir }}
-
+    [[ -d {{ custom_vars.output_dir }} ]] || sudo mkdir -m a+w {{ custom_vars.output_dir }}
+  args:
+    executable: /bin/bash
 - name: Write file for Slurm machines
   with_items: "{{ custom_vars.partitions }}"
   ansible.builtin.shell: |
     srun -N 1 -p '{{ item }}' bash -c 'echo {{ item }} > {{ custom_vars.output_dir }}/{{ item }}.txt'
+  args:
+    executable: /bin/bash
   register: write_file_output
 
 - name: Print console output
   ansible.builtin.debug:
     msg: "{{ write_file_output }}"
 
-- name: Ensure a file exists for each OS type
+- name: Ensure a file exists for each partition
   with_items: "{{ custom_vars.partitions }}"
   ansible.builtin.shell: |
-    test $(cat {{ custom_vars.output_dir }}/{{ item }}.txt) == '{{ item }}' && echo 'Success!' || exit 1
+    test $(cat {{ custom_vars.output_dir }}/{{ item }}.txt) == '{{ item }}' || exit 1
```

Comment:
Sorry to nitpick here, but `test "a" == "b"` already exits 1. Unless this is doing something valuable that I'm not catching onto, please remove it.

Reply:
Oh I see, sorry I misunderstood the first time.

Reply:
Fixed.